Sprite.java

```diff
@@ -315,4 +317,53 @@ private void processScripts(JSONArray array)
 
         return;
     }
+
+    /**
+     * Get list of Variables.
+     *
+     * @return variables
+     */
+    public String[] getVariables()
+    {
+        return variables;
+    }
+
+    /**
+     * Populate list of variables.
+     */
+    public void populateVariables()
+    {
+        JSONArray vars = 
+            FileUtils.getJSONArrayAttribute(this.jsonObj, "variables");
```

Comment:
'this' keyword is not necessary here - since we're only referring to a single jsonObj field, just 'jsonObj' is sufficient.